tests/payment/Payment - Credit card and Food Voucher - Finish Purchase - ambienteqa.test.js

```diff
@@ -1,3 +1,3 @@
 import test from './models/Payment - Credit card and Food Voucher - Finish Purchase.model'
 
-test('ambienteqa')
```

Comment:
you'll need to rename this file to `Payment - Credit card and Food Voucher - Finish Purchase - gatewayqa.test.js`